### PR TITLE
Design Fixes

### LINKED ISSUE #10: [Main] Design Feedback
@paperpsych to add feedback here

### LINKED ISSUE #62: [design] CTA highlight
The 2nd CTA of the banner needs a bit more highlight—colors/solid border or something else.

image

### PR DESCRIPTION
**Pull request by emjshrx**
fixes #62 fixes #10 

Screenshot 2023-12-06 at 12 37 30 AM

Screenshot 2023-12-06 at 12 20 50 AM

### CODE REVIEW

**Comment by keraliss** [comment]
The padding is looking way too big in my screen, is this my device issue? or it is intended that way? also can you please add some padding in the mobile version? px-2 i guess?

**Comment by keraliss** [comment]
also the previous menu, without the sticky thing was looking better in my eyes. @rajarshimaitra  @setusaurabh any opinion? can't find paperpsyche here!

File: src/pages/index.astro [hunk 1]
```diff
@@ -39,12 +39,12 @@ import Header from "../components/common/Header.astro";
         <a
           href="https://discord.gg/ekzAUeeR"
           target="_blank"
-          class="text-m mx-3 flex w-1/2 items-center justify-center rounded-lg bg-orange p-4 text-center lg:text-2xl"
+          class="text-m mx-3 flex w-1/2 items-center justify-center rounded-lg bg-orange p-4 text-center hover:bg-black hover:text-orange hover:outline-dashed lg:text-2xl"
           >Dive into the Rabbit Hole</a
```
Comment (keraliss) [inline src/pages/index.astro:43]: font-semibold maybe? would look better i think

**Comment by rajarshimaitra** [comment]
> also the previous menu, without the sticky thing was looking better in my eyes.

I would prefer the menu bar to be sticky.. Feels easier to navigate..  But that's just me..

**Comment by rajarshimaitra** [comment]
@emjshrx is it possible to make the 2nd CTA on the banner look more buttonish?

**Comment by emjshrx** [comment]
> The padding is looking way too big in my screen, is this my device issue? or it is intended that way? also can you please add some padding in the mobile version? px-2 i guess?

Yes on large screens the padding is intended as it was becoming difficult to read as we have to scan across the screen for a single line. Will add padding on mobile to few components. Images are supposed to be without padding in mobile according to the design.

**Review by keraliss — APPROVED** [review]

**Pull request merged**